sdk/storage/azure-storage-blob/pom.xml

```diff
@@ -52,11 +52,6 @@
   </pluginRepositories>
 
   <dependencies>
-    <dependency>
```

Comment:
This shouldn't be deleted as azure-core-test may bring in an older version of azure-core.